src/LandingPage.js

```diff
@@ -6,6 +6,7 @@ class LandingPage extends Component {
   constructor(props) {
     super(props);
     this.state = {
+      backgroundImg: null
```

Comment:
Does LandPage need this property? I'm not seeing where it's used, but I might be missing something